### PR TITLE
ekg2: patch added

### LINKED ISSUE #41268: ekg2 fails to build on 10.10+
I installed *all* formulae from the core, this is one of the issues I ran into. I’m at work, will check tonight.

```
$ brew install ekg2
==> Downloading http://pl.ekg2.org/ekg2-0.3.1.tar.gz
######################################################################## 100,0%
==> ./configure --prefix=/usr/local/Cellar/ekg2/0.3.1 --without-python --without-perl --with-readlin
==> make install
3 errors generated.
make[2]: *** [rc_la-inputs.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [install-recursive] Error 1
make: *** [install-recursive] Error 1

READ THIS: https://git.io/brew-troubleshooting
```

Logs: https://gist.github.com/a94ebff10a0f6d10ede3

### PR DESCRIPTION
**Pull request by bfontaine**
This fixes #41268.

### CODE REVIEW

**Comment by DomT4** [comment]
Can we revision if this previously linked the system OpenSSL please? :)

**Comment by bfontaine** [comment]
Sure :)

File: Library/Formula/ekg2.rb [hunk 1]
```diff
@@ -1,28 +1,115 @@
...
+  revision 1
 
   depends_on "pkg-config" => :build
   depends_on "readline"
   depends_on "libgadu" => :optional
+  depends_on "openssl"
+
+  # Fix the build on OS X 10.9+
+  # http://bugs.ekg2.org/issues/152
+  patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/ekg2.rb]: The Gist link there doesn't work CC @dunn.

**Comment by mikemcquaid** [comment]
:+1: once we've figured out the patch stuff.

File: Library/Formula/ekg2.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/ekg2.rb]: :see_no_evil: that's what I get for pruning my gists.

Comment (dunn) [inline Library/Formula/ekg2.rb]: Forgot my password for their issue tracker; will add a new gist once I get that sorted.

Comment (bfontaine) [inline Library/Formula/ekg2.rb]: :closed_lock_with_key:

**Comment by bfontaine** [comment]
I wrote a comment on the bug report with a working gist URL.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by dunn** [comment]
Ahh, thanks!

**Pull request closed without merging**